### PR TITLE
Added Crystal Network

### PR DESCRIPTION
**Pull request by ziuee**
### General:
* [ ] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [ ] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [ ] No changes were made to the file's formatting.
* [ ] There are no syntax errors.
* [ ] There is no pre-existing mapping matching my id.
* [ ] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [ ] `id`: a lowercase string *(ex. `myserver`)*
  - [ ] `name`: a string *(ex. `MyServerPvP`)*
  - [ ] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:
* [ ] My image is a `png` file.
* [ ] I uploaded my image to the `logos` directory.
* [ ] My image has a transparent background and is square (1:1 aspect ratio).
* [ ] My image is `512` pixels in width and height.
* [ ] My image's file name matches its mapping id.

### CODE REVIEW

**Comment by Spoony1337** [comment]
This guy is gay, this request should be denied

**Comment by 128931** [comment]
skidders are haram this should be closed asap

File: servers.json [hunk 1]
```diff
@@ -366,6 +366,13 @@
             "shotbow.net"
         ]
     },
+    {
+        "id": "crystal",
+        "name": "Crystal Network",
+        "addresses": [
+            "crystal.rip",
```
Comment (128931) [inline servers.json:373]: Skid can't comprehend how JSON works and adds a comma there 😂😂😂

**Comment by Scifi9902** [comment]
Would heavily recommend not accepting this PR.

File: servers.json (hunk 1 above)
Comment (ziuee) [reply servers.json:373]: Simply just didn’t get added in the json by accident

**Comment by ziuee** [comment]
> Would heavily recommend not accepting this PR.

What for? Looks like the Latino community is a little mad cus I exposed y’all for skidding now tryna reverse it back to me with 0 proof lmfaoo

**Comment by ziuee** [comment]
Also what in the heck are you on about with invalid formatting?

**Comment by LinuxDino** [comment]
Let this be closed :)

**Pull request closed without merging**